.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+repos:
+  -   repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v3.2.0
+      hooks:
+        -   id: trailing-whitespace
+        -   id: end-of-file-fixer
+        -   id: check-yaml
+        -   id: check-added-large-files
+  -   repo: https://github.com/pre-commit/mirrors-eslint
+      rev: ''  # Use the sha / tag you want to point at
+      hooks:
+        - id: eslint
+          additional_dependencies:
+          -   eslint@8.32.0
+          -   eslint-config-prettier@8.6.0
+          -   eslint-plugin-prettier@4.2.1
+          -   eslint-plugin-react@7.32.1
```

Comment:
Added this to include javascript and typescript files (per [this documentation](https://github.com/pre-commit/mirrors-eslint)). May want to expand as project grows.

```yaml
files: \.[jt]sx?$  # *.js, *.jsx, *.ts and *.tsx
types: [file]
```